lib/src/mt_ptp.c

```diff
@@ -92,7 +91,7 @@ static inline int ptp_timesync_read_tx_time(struct mt_ptp_impl* ptp, uint64_t* t
   ret = rte_eth_timesync_read_tx_timestamp(port_id, &spec);
   ptp_timesync_unlock(ptp);
 
-  if (ret < 0) err("%s(%d), err %d\n", __func__, port, ret);
+  // if (ret < 0) err("%s(%d), err %d\n", __func__, port, ret);
```

Comment:
Why remove this error msg?

Reply:
> Why remove this error msg?

Because the first few reading tx timestamp attempts always failed, we have set total 50 attempts to tolerate the fails. So, it is unnecessary to print error messages of the first few attempts. Error message is required only when the number of attempts is larger than 50.

Reply:
ok, pls help to set the level to dbg